libs/site-prismic/src/slices/ComparisonBlock/index.module.css

```
@@ -0,0 +1,51 @@
+
+.comparisonBlock {
+    background-color: color-mix(in srgb, var(--thecoin-gray2) 50%, white);
+    border-bottom: 1px solid var(--text-dark-color);
+    box-shadow: 0px 2px 5px 0px var(--thecoin-gray3);
+
+    margin: 0 auto;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)